src/main/kotlin/com/appland/appmap/AppmapPlugin.kt

```diff
@@ -0,0 +1,22 @@
+package com.appland.appmap
```

Comment:
The plugin is best seen as a kotlin class, I found it to be closer to the general work with gradle so I used it instead of Java. @dustinbyrne

Reply:
now the plugin is working with java code.